manifests/init.pp

```diff
@@ -325,9 +329,11 @@
       register_in_foreman   => $register_in_foreman,
       foreman_base_url      => $foreman_url,
       registered_proxy_url  => "https://${capsule_fqdn}:${capsule::foreman_proxy_port}",
+      trusted_hosts         => undef,
```

Comment:
Sorry? This isn't fixing 9102, and 9102 isn't going to get fixed anytime soon.

This is *required* to use the templates plugin today, you need to disable trusted hosts...and thats what this line is doing.